docs/ref/forms/api.txt

```diff
@@ -1499,12 +1521,57 @@ be implemented as follows::
 
 
     class GPSCoordinatesField(Field):
-        def get_bound_field(self, form, field_name):
-            return GPSCoordinatesBoundField(form, self, field_name)
+        bound_field_class = GPSCoordinatesBoundField
 
 Now you can access the country in a template with
 ``{{ form.coordinates.country }}``.
 
+You may also want to customize the default form field template rendering. For
+example, you can override :meth:`.BoundField.label_tag` to add a custom class::
+
+   class StyledLabelBoundField(BoundField):
+       def label_tag(self, contents=None, attrs=None, label_suffix=None, tag=None):
+           attrs = attrs or {}
+           attrs["class"] = "wide"
+           return super().label_tag(contents, attrs, label_suffix, tag)
+
+
+   class UserForm(forms.Form):
+       bound_field_class = StyledLabelBoundField
+       name = CharField()
+
+This would update the default form rendering:
+
+.. code-block:: pycon
+
+   >>> f = UserForm()
+   >>> print(f["name"].label_tag)
+   <label for="id_name" class="wide">Name:</label>
+
+Another requirement could be that you wanted to add a CSS class to the wrapping
+HTML element of all fields. You could easily achieve this by defining
+additional CSS classes as follows::
```

Comment:
minor: This feels a bit wordy to me. I wonder if it would be improved by stating something like

```suggestion
To add a CSS class to the wrapping HTML element of all fields, a `BoundField` can be 
overridden to return a different collection of css classes::
```

My reasoning is that some new developers may not know what a "requirement" means because they aren't in a software job. However, the use case of adding a "css class to the wrapping html" element should be common (if someone is reading the docs for that).

Reply:
Agree, also just clearer to use fewer words.

Reply:
Thanks. I stumbled around formulating this, your version sounds much clearer.